iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/VariantManagerImplTest.java

```diff
@@ -57,11 +57,36 @@ public boolean hasProperty() {
             return false;
         }
 
+        @Override
+        public boolean hasProperty(String key) {
+            return false;
+        }
+
         @Override
         public Properties getProperties() {
             return null;
         }
```

Comment:
Do not override this method: the default behaviour is given by the interface (default method)